pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -729,3 +730,47 @@ func generateServerURL(serverIP string, nodePort int32) (string, error) {
 func SupportedStorageMode() []string {
 	return []string{etcdStorageModeEmptyDir, etcdStorageModeHostPath, etcdStorageModePVC}
 }
+
+// NewDefaultCommandInitOption returns a CommandInitOption with default values
+func NewDefaultCommandInitOption() *CommandInitOption {
+	return &CommandInitOption{
+		ImagePullPolicy:                    string(corev1.PullIfNotPresent),
+		KubeImageTag:                       "v1.29.6",
+		Namespace:                          "karmada-system",
```

Comment:
does we have not constants for it?

Reply:
I was not observed in the search system, which has the corresponding constants, in NewDefaultCommandInitOption approach is to write, I mainly refer to is the PKG/karmadactl/cmdinit/cmdinit. Go in for the default value of cmdflag, there are Parts of const variables are mostly copied using const constants

Will need these for example KubeImageTag string constant changes in the form of variables on the PKG/karmadactl/cmdinit/kubernets/deploy. Go in?

Reply:
> Will need these for example KubeImageTag string constant changes in the form of variables on the PKG/karmadactl/cmdinit/kubernets/deploy. Go in?

How about creating constants like `KubeImageTag` and `KarmadaNamespace` to reduce shotgun surgery modifications?

Reply:
> > Will need these for example KubeImageTag string constant changes in the form of variables on the PKG/karmadactl/cmdinit/kubernets/deploy. Go in?将需要将这些例如 KubeImageTag 字符串常量以变量的形式放在 PKG/karmadactl/cmdinit/kubernets/deploy 上。进去？
> 
> How about creating constants like `KubeImageTag` and `KarmadaNamespace` to reduce shotgun surgery modifications?创建像 `KubeImageTag` 和 `KarmadaNamespace` 这样的常量来减少霰弹枪手术修改怎么样？

Ok, I'll add some const variables to the config package and modify the code